src/main.rs

```diff
@@ -884,9 +891,9 @@ async fn get_interactive(get: GetInteractive, out: Option<PathBuf>) -> Result<()
                 // Rename temp file, to target name
                 let filepath2 = filepath.clone();
                 if let Some(parent) = filepath2.parent() {
-                    tokio::fs::create_dir_all(parent)
-                        .await
-                        .context("Unable to create directory {parent}")?;
+                    tokio::fs::create_dir_all(parent).await.with_context(|| {
```

Comment:
with_context is so that the formatting is lazy?

Reply:
The previous version did not do any formatting, just literally printed `{parent}` in the output.

`.with_context` is if you need to do any anything that's not a static `Context` with a `Display`.  `format!` is just the most common version of this.

Reply:
oh, ok...